package.json

```diff
@@ -84,12 +83,12 @@
     "test:watch": "npm test -- --watch",
     "test:examples": "node examples/",
     "cover": "cross-env BABEL_ENV=commonjs istanbul cover _mocha -- --compilers js:babel-register --recursive",
-    "lint": "eslint src test --fix",
+    "lint": "eslint src test",
     "lint:fix": "npm run lint -- --fix",
     "build": "cross-env BABEL_ENV=commonjs babel src --out-dir lib --source-maps",
     "flow": "flow",
     "prettier": "prettier --write ./**/*.{js,css,md}",
-    "prepublish": "npm run clean && npm run lint && npm run build && npm run test"
```

Comment:
Saw a deprecation warning for `prepublish`, swapped for `prepare` to maintain similar behavior

```
       o prepublish: Run BEFORE the package is packed and published, as well as on local npm install without any
         arguments. (See below)

       o prepare: Run both BEFORE the package is packed and published, on local npm install  without  any  argu-
         ments,  and when installing git dependencies (See below). This is run AFTER prepublish, but BEFORE pre-
         publishOnly.
```

Reply:
mmmm...yes, thank you